### ISSUE TITLE
Transfer libra from one acc to another

### ISSUE THREAD

**Issue by kenken64** [perfectmak/libra-core#18]
Hi,

What is the best way to xfer libra from one acc to another without creating a new account?

I cannot find a function that returns an account object based on the address 

const account = wallet.newAccount();
  const account2Address = '854563c50d20788fb6c11fac1010b553d722edb0c02f87c2edbdd3923726d13f';
  const response = await client.transferCoins(account, account2Address, 1e6);

**Comment by kenken64** [comment]
does it mean if I use this passing in the secret key I can get an Account object?

` public static fromSecretKey(secretKeyHex: string): Account {
    const keyBytes = new Uint8Array(Buffer.from(secretKeyHex, 'hex'));
    return Account.fromSecretKeyBytes(keyBytes);
  }`

**Comment by totiz** [comment]
@kenken64 actually  ```newAccount()``` doesn't create a brand new account everytime, it just generate account from DerivedPath, and DerivedPath is the form like this ```LIBRA WALLET: derived key${childDepth}```. So if you back up your mnemonic phrase, next time when using the same mnemonic you'll always access to the same account.

For example
```javascript
const {LibraWallet} = require('libra-core')

wallet = new LibraWallet({
  mnemonic: 'retire addict oppose canal spy table junior fade lock chief try cream eyebrow obscure grunt install define bread absorb chief toddler either audit fine'
})

const account = wallet.newAccount()

console.log(account.getAddress().toHex())
```
You will get  **469bb2d891eb89d7a4689ab7c8c1a1f3220918c6fd7191acfa29be9f2931477c**.

Or you can use ```generateAccount(depth: number)``` for less confuse.
```javascript
const firstAccount = wallet.generateAccount(0)
const secondAccount = wallet.generateAccount(1)

console.log(`first account address is ${firstAccount.getAddress().toHex()}`)
console.log(`second account address is ${secondAccount.getAddress().toHex()}`)
```

**Comment by kenken64** [comment]
thanks!

**Comment by perfectmak** [comment]
@kenken64 Thank you for using the library. I'll close this now. Please open an issue if you experience any more troubles. Also there is a new fix that has been released as version [1.0.4](https://github.com/perfectmak/libra-core/releases/tag/1.0.4). It fixes the non backward compatible changes, but ideally, you shouldn't need to change anything in your code.
